content/docs/card/card-comment.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Card Comment
+description: skeleton of comment card
+author: YourTwitterUsername
+---
+
+<ComponentPreview name="card-card-comment--docs" />
+
+## Installation
+
+<Steps>
+<Step>Install dependencies</Step>
+
+```bash
+npm install framer-motion lucide-react
+```
+
+<Step>Update `tailwind.config.js`</Step>
+
+Add the following to your tailwind.config.js file.
+
+```json
+module.exports = {
+  theme: {
+    extend: {
+    }
+  }
+}
+```
+
```

Comment:
@Abishkardhenga you can get rid of these as you don't need these two steps.

Reply:
yeah just remove line no 18 and 20  too .

Reply:
<img width="1440" alt="Screenshot 2024-10-02 at 00 47 27" src="https://github.com/user-attachments/assets/f4645f9d-a67c-485d-8b53-9e955b58c862">
 this is the reason so i remove